docs/user/helper.rst

```diff
@@ -16,3 +17,140 @@ length of the arrays and not the number of keys, and you get a
+         works with classification tasks.
+       --select__k : int or "all", optional, default=10
+         Number of top features to select.
+         The "all" option bypasses selection, for use in a parameter search.
+
+    ...
+
+    <NeuralNetClassifier> options:
+       --net__module : torch module (class or instance)
+         A PyTorch :class:`~torch.nn.Module`. In general, the
+         uninstantiated class should be passed, although instantiated
+         modules will also work.
+       --net__criterion : torch criterion (class, default=torch.nn.NLLLoss)
+         Negative log likelihood loss. Note that the module should return
+         probabilities, the log is applied during ``get_loss``.
+       --net__optimizer : torch optim (class, default=torch.optim.SGD)
+         The uninitialized optimizer (update rule) used to optimize the
+         module
+       --net__lr : float (default=0.01)
+         Learning rate passed to the optimizer. You may use ``lr`` instead
+         of using ``optimizer__lr``, which would result in the same outcome.
+       --net__max_epochs : int (default=10)
+         The number of epochs to train for each ``fit`` call. Note that you
+         may keyboard-interrupt training at any time.
+       --net__batch_size : int (default=128)
+         ...
+       --net__verbose : int (default=1)
+         Control the verbosity level.
+       --net__device : str, torch.device (default='cpu')
+         The compute device to be used. If set to 'cuda', data in torch
+         tensors will be pushed to cuda tensors before being sent to the
+         module.
+
+    <MLPClassifier> options:
+       --net__module__hidden_units : int (default=10)
+         Number of units in hidden layers.
+       --net__module__num_hidden : int (default=1)
+         Number of hidden layers.
+       --net__module__nonlin : torch.nn.Module instance (default=torch.nn.ReLU())
+         Non-linearity to apply after hidden layers.
+       --net__module__dropout : float (default=0)
+         Dropout rate. Dropout is applied between layers.
+
+Installation
+^^^^^^^^^^^^
+
+To use this functionaliy, you need some further libraries that are not
+part of skorch, namely fire_ and numpydoc_. You can install them
+thusly:
+
+
+.. code:: bash
+
+    pip install fire numpydoc
+
+Usage
+^^^^^
+
+When you write your own script, only the following bits need to be
+added:
+
+.. code:: python
+
+    import fire
+    from skorch.helper import parse_args
+
+    # your model definition and data fetching code below
+    ...
+
+    def main(**kwargs):
+        X, y = get_data()
+        my_model = get_model()
+
+        # important: wrap the model with the parsed arguments
+        parsed = parse_args(kwargs)
+        my_model = parsed(my_model)
+
+        my_model.fit(X, y)
+
+
+    if __name__ == '__main__':
+        fire.Fire(main)
+
+
+This even works if your neural net is part of an sklearn pipeline, in
```

Comment:
```suggestion
This even works if your neural net is part of a sklearn pipeline, in
```

Reply:
I pronounce it "ess-kay-learn", therefore I would use "an". Or do most people "translate" it to "scy-kit-learn"?